include/treelite/tree.h

```diff
@@ -264,6 +266,59 @@ class Tree {
     /* \brief whether the list given by MatchingCategories(nid) is associated with the right child
      *        node or the left child node. True if the right child, False otherwise */
     bool categories_list_right_child_;
+
+    /** Getters **/
+    inline int LeftChild() const {
+      return cleft_;
+    }
+    inline int RightChild() const {
+      return cright_;
+    }
+    inline bool DefaultLeft() const {
+      return (sindex_ >> 31U) != 0;
+    }
+    inline int DefaultChild() const {
+      return ((sindex_ >> 31U) != 0) ? cleft_ : cright_;
```

Comment:
Could you also add comments for this and the other bitwise manipulations in here?

Reply:
Done